### PR TITLE
NakeHeader

### PR DESCRIPTION
**Pull request by YehorHeiko**
- [DEMO LINK](https://YehorHeiko.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://YehorHeiko.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -7,8 +7,46 @@
...
+            <a href="#" class="nav__link">Tablets</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Photo</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Video</a>
+          </li>
+        </ul>
+      </nav>
```
Comment (anastasiia-tilikina) [inline src/index.html:49]: image

pls follow style guide

File: src/index.html [hunk 2]
```diff
@@ -7,8 +7,46 @@
...
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="header__logo" href="#">
+        <img class="header__img" src="./images/logo.png"
+        alt="logo">
```
Comment (anastasiia-tilikina) [inline src/index.html:19]: image

same here

File: src/style.css [hunk 3]
```diff
@@ -1,3 +1,59 @@
...
+      font-family: Roboto, sans-serif;
+      color: #000;
+      font-size: 12px;
+      font-weight: 500;
+      line-height: normal;
+      text-transform: uppercase;
+      text-decoration: none;
+      padding: 23px 0;
+      display: flex;
+}
```
Comment (anastasiia-tilikina) [inline src/style.css:33]: fix indentation

File: src/style.css [hunk 4]
```diff
@@ -1,3 +1,59 @@
...
+.is-active,
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.header__img {
+  width: 40px;
+  height: 40px;
+  flex-shrink: 0;
+  margin-top: 5px;
```
Comment (anastasiia-tilikina) [inline src/style.css:44]:
```suggestion
```

```css
header__logo {
  display: inline-flex;
}
``` 

this option would be better for positoning logo

**Review by anastasiia-tilikina — CHANGES_REQUESTED** [review]
GJ

check my comments

File: src/style.css (hunk 4 above)
Comment (YehorHeiko) [reply src/style.css]: Fixed

File: src/index.html [hunk 5]
```diff
@@ -7,8 +7,49 @@
...
+          </li>
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#smartphones" class="nav__link">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a href="#laptops_&_computers" class="nav__link"
+            data-qa="hover"> Laptops & Computers </a>
```
Comment (loralevitska) [inline src/index.html:37]:
```suggestion
            <a 
              href="#laptops_&_computers" 
              class="nav__link"
              data-qa="hover"
            >
              Laptops & Computers
            </a>
```

**Review by loralevitska — APPROVED** [review]
Good job! Check my comments for improvement 
- still not fixed 
image

**Pull request closed without merging**